chart/values.yaml

```diff
@@ -48,6 +48,59 @@ kube:
 # Set to true to enable support for multiple availability zones.
 multi_az: false
 
+# Set the availability zones when multi_az is set to true.
+# E.g
+# api:
+#   azs:
+#   - us-south-1
+#   - us-south-2
+availability_zones:
```

Comment:
Do we need the defaults here?
Would be nice to just provide a sample and keep `values.yaml` smaller.

Reply:
@viovanov I have updated the `values.yaml` file and provide a sample for availability_zones.